modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/message/SqlQueryMessageGroup.java

```diff
@@ -40,4 +40,10 @@ public final class SqlQueryMessageGroup {
     public static final short QUERY_BATCH_REQUEST = 4;
 
     public static final short QUERY_CLOSE_MESSAGE = 5;
+
+    /** See {@link CancelOperationRequest} for the details. */
+    public static final short OPERATION_CANCEL_REQUEST = 100;
```

Comment:
Separate codes are used because I assumed that these messages will move to a separate message group in the future. If this assumption is incorrect, it should be changed to have continuous numbering.

Reply:
I thought that since they are not related to the execution of SQL queries (and are not executed through the `MessageService` introduced in the sql-engine module) it is better to put them into a separate group.
But overall it probably doesn't matter, so I added the continuous numbering